target/linux/lantiq/image/Makefile

```diff
@@ -147,9 +150,10 @@ endef
 TARGET_DEVICES += arcadyan_arv4520pw
 
 define Device/arcadyan_arv4525pw
+  DEVICE_VENDOR := Arcadyan
+  DEVICE_MODEL := Speedport W502V Typ A - ARV4525PW
```

Comment:
Same here, consider `Speedport W502V Typ A (ARV4525PW)`
(This is just a subjective suggestion, though)